source/docs/advanced/graphql/slim-down-resolvers-with-loaders.md

```diff
@@ -316,37 +320,29 @@ Let’s now take a look at a _real-world_ module definition as an example.
 Front-Commerce itself is written as GraphQL modules. You can browse
 Front-Commerce’s source code to find more examples and understand how it works.
 
-Here is for instance how Magento2 CMS module’s definition looks like. You should
+Here is for instance how Magento2 RequisitionLists module’s definition looks like. You should
 see several patterns mentioned previously and get ideas about applying them in
 your application:
 
 ```js
-import {
-  makeUserClientFromRequest
-} from "server/modules/magento2/core/factories";
+import { makeUserGraphQLClientFromRequest } from "server/modules/magento2/core/factories";
 
 import typeDefs from "./schema.gql";
-import resolvers from "./resolvers";
-import { CmsBlockLoader, CmsPageLoader } from "./loaders";
+import resolvers from "./resolver";
+import RequisitionListLoader from "./loaders/RequisitionListLoader";
```

Comment:
I think we should keep in the example types of the core fc as they are more familiar to the integrators. requisition list is part of the B2B and most will not know about it and this might through them off a bit i think

Reply:
I agree, but as far as I know, there is no loaders with the new usage in the core yet.

This PR might wait until we have such an example to use.